### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jerpint**
* Ship data with example app
* Update readme with better instructions
* add CLI args for some tool utilites

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,89 @@
...
+This will launch the gradio app locally.
 
-    cd buster/buster/examples
+# Generating your own embeddings
 
-We've attached a sample `stackoverflow.csv` file to help you get started.
+Once your local version of Buster is up and running, the next step is for you to be able to import your own data.
+We will be using the `stackoverflow.csv` file in the `buster/examples/` folder for this. This is the same data that was used to generate the demo app's embeddings.
 
 You will first ingest the documents to be ready for buster. In this example, we use Deeplake's vectore store, but you can always write your own custom `DocumentManager`:
```
Comment (hbertrand) [inline README.md:51]: typo: vectore -> vector

File: README.md [hunk 2]
```diff
@@ -1,71 +1,89 @@
...
+**Note**: Buster requires python>=3.10
 
-```
+```bash
 pip install buster-doctalk
 ```
 
-If you would like to install a local version, run this instead:
+Then, go to the examples folder and launch the app.
+We've included small sample data on stackoverflow-ai questions that you can test your setup with to try app:
```
Comment (hbertrand) [inline README.md:31]: typo: small sample data on stackoverflow-ai questions -> small sample data off stackoverflow-ai questions

**Review by hbertrand — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jerpint) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (jerpint) [reply README.md]: fixed

**Pull request merged**